specifications/sep-payment-channel-transactions.md

```diff
@@ -565,6 +568,24 @@ process if its payment operation is dependent on amounts frozen.
 There is nothing participants can do to prevent this, other than using only auth
 immutable assets.
 
+### Trustline Limits
+
+Trustlines on the escrow accounts are defined as always having a maximum asset
+limit. This restriction makes the behavior of the closing transaction as
+predictable as possible and simplifies implementations that are designed for
+operating on common assets that do not have excessive supply.
+
+Implementations that allow lower asset limits may produce closing transactions
```

Comment:
my confusion arises from this part. The closing transaction will be paid as a claimable balance so therefore should not be able to fail as a result of the destination account's trustline limit, correct?

Reply:
Ah, so we don't use claimable balances. The closing transaction contains these operations:
https://github.com/stellar/experimental-payment-channels/blob/f421bace8d0917a9fd7e7ae9935f492a158bda62/specifications/sep-payment-channel-transactions.md#L264-L270

Reply:
We didn't use claimable balances because they require sponsorship. Creation of new ledger entries can fail if sponsorship fails which makes the protocol more complicated. It requires us to store reserves in accounts. It resulted in us needing two escrow accounts so that the accounts are completely separated at close, because otherwise reserves and sponsorship mixing could result in the accounts being bound together after close. Creating a claimable balance is a new ledger entry which is why reserves/sponsorship are involved.

So instead we we use simple payments between two escrow accounts. 🎉